service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -81,7 +81,12 @@ void shouldAddCaseLocalAuthorityToCaseData() throws Exception {
             .andExpect(status().isOk())
             .andReturn();
 
-        assertThat(response.getResponse().getContentAsString()).contains(expectedData.toString());
+        AboutToStartOrSubmitCallbackResponse callbackResponse = MAPPER.readValue(response.getResponse()
+            .getContentAsByteArray(), AboutToStartOrSubmitCallbackResponse.class);
```

Comment:
Since you have done it please remove 

```
JSONObject expectedData = new JSONObject();
         expectedData.put("caseName", "title");		         expectedData.put("caseName", "title");
         expectedData.put("caseLocalAuthority", "EX");		         expectedData.put("caseLocalAuthority", "EX");
```